CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ option(kseta_documentation "Whether or not to create doxygen doc target.")
 include(CTest)
 
 #--- enable CPack --------------------------------------------------------------
-include(cmake/ksetaCPack.cmake)
+#include(cmake/ksetaCPack.cmake)
```

Comment:
Do we really would like to remove it for all for now?
Maybe better add an issue first...

Reply:
Opened issue #4 to keep track of this.